sklearn/tests/test_pipeline.py

```diff
@@ -672,15 +680,15 @@ def make():
     assert_array_equal([exp], pipeline.fit(X).predict(X))
     assert_array_equal(X, pipeline.inverse_transform([[exp]]))
     assert (pipeline.get_params(deep=True) ==
-                      {'steps': pipeline.steps,
-                       'm2': mult2,
-                       'm3': passthrough,
-                       'last': mult5,
-                       'memory': None,
-                       'm2__mult': 2,
-                       'last__mult': 5,
-                       'verbose': False
-                       })
+            {'steps': pipeline.steps,
```

Comment:
Why all the unrelated changes?

(I don't mind but we usually tell contributors to avoid those)

Reply:
I think I fixed pep8 in two places. Here and the indentationabove. The pytest.raises are required changes.